### PR TITLE
Update login page copy

### PR DESCRIPTION
**Pull request by mrchrisadams**
This should address the following trello card, about asking for help.

https://trello.com/c/voI0Bskm/9-update-contact-info-log-in-page

This introduces:

- updated email copy
- update front page copy
- tests for the component

### CODE REVIEW

**Comment by mrchrisadams** [comment]
I'm having a hard time with the validation library, which is causing an error in the tests. I see the functionality working in manual testing, but this test is failing and I'm wary of it taking up loads of time:

```javascript
test('the submit button should no longer be disabled with valid data', async () => {

          // arrange - allowing the form to render after 
          await wrapper.vm.$nextTick()
 
         // set the value
          wrapper.find("input[name='email']").setValue("valid.email@domain.com")

          // for reasons which aren't obvious we need to set the data too. I think this is related to vee-validate
         // wrapper.setData({email: "valid.email@domain.com"})

        // Because validation occurs on blur we need to trigger the blur, then wait for the next 'tick'
          wrapper.find("input[name='email']").trigger('blur')

          await wrapper.vm.$nextTick()

          // assert that the button is on now
          expect(wrapper.find('button').element.disabled).toBe(false)
        })

    })

```

**Comment by mrchrisadams** [comment]
Ah, maybe this is the issue. I assumed the call to next tick would resolve all promises but this might not be the case, based on [this info in the docs for Vee Validate](https://logaretm.github.io/vee-validate/advanced/testing.html#asynchronous-testing)

> After triggering an event like an input event, make sure to call flushPromises before checking the UI for changes:

**Comment by mrchrisadams** [comment]
I've done some research for the the validation issue, and it looks like it's quite a rabbit hole to go down.

As such, I've summarised the issue, so we can at least merge in the copy changes.
https://github.com/Greening-Digital/constellate/issues/72

**Comment by mrchrisadams** [comment]
@rmparr can I get a quick review of this mate?

File: backend/backend/templates/passwordless_default_token_email.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+</head>
+<body>
+    <h2>Use this code to log in: {{ callback_token }}</h2>
+    <h3>Note: this login code is valid for the next 15 minutes, and can only be used once.</h3>
+
+
+
+
+</body>
+</html>
```
Comment (rmparr) [inline backend/backend/templates/passwordless_default_token_email.html:15]: Missing the trailing newline.

File: cl8-web/src/components/auth/Login.vue [hunk 2]
```diff
@@ -59,12 +61,12 @@
               <div class="w-100">
                 <input
                   type="password"
-                  name="token"
+                  name="Login Code"
```
Comment (rmparr) [inline cl8-web/src/components/auth/Login.vue:64]: I think input names are usually kebab-case.

**Pull request merged**